draft-wing-cidfi.md

```diff
@@ -1234,6 +1234,22 @@ Register new special-use domain name cidfi.arpa for DNS SVCB discovery.
 This document requests IANA to register the new DNS SVCB "_cidfi-aware" in
 the "DNS Service Bindings (SVCB)" registry available at {{IANA-SVCB}}.
 
+The document also requests IANA to register the following service parameter
```

Comment:
We will also have to add cidfipathauth and cidfimetadata.

Reply:
We will need to consider the sync the attributes between the various registries but we also need to make sure that we are not overloading SVCB. Will create an issue for this + not implement more changes as part of this PR.